### ISSUE TITLE
[p5.js 2.0+ Bug Report]: Missing type declarations

### ISSUE THREAD

**Issue by m4lt3** [processing/p5.js#9190]
### Most appropriate sub-area of p5.js?

- [x] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [x] Utilities
- [ ] WebGL
- [ ] WebGPU
- [ ] p5.strands
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [ ] Other (specify if possible)

### p5.js version

2.3.3

### Steps:
1. install p5js via `npm i p5`
2. create a `sketch.ts` file and add `import p5 from "p5"` or `import "p5/global"`
3. Get the error `Could not find a declaration file for module 'p5'`
4. look in the package.json to see there are `types` declared
5. open the `p5` folder in `node_modules` and find the files missing
6. Surprised Pikachu Face

### Snippet:

```ts

import p5 from "p5";

```
